scipy/special/_ndtri_exp.pxd

```diff
@@ -0,0 +1,174 @@
+"""
+Implementation of the inverse of the logarithm of the CDF of the standard
+normal distribution.
+
+Copyright: Albert Steppi
+
+Distributed under the same license as SciPy
+
+
+Implementation Overview
+
+The inverse of the CDF of the standard normal distribution is available
+in scipy through the Cephes Math Library where it is called ndtri.
+We call our implementation of the inverse of the log CDF ndtri_exp.
+For -2 <= y <= log(1 - exp(-2)),  ndtri_exp is computed as ndtri(exp(y)).
+
+For 0 < p < exp(-2), the cephes implementation of ndtri uses an approximation
+for ndtri(p) which is a function of z = sqrt(-2.0 * log(p)). Letting
+y = log(p), for y < -2, ndtri_exp uses this approximation in log(p) directly.
+This allows this implementation to achieve high precision for very small y,
+whereas ndtri(exp(y)) becomes inf when exp(y) underflows for y < ~ -745.1.
```

Comment:
```suggestion
This allows the implementation to achieve high precision for very small y,
whereas ndtri(exp(y)) evaluates to infinity. This is because exp(y) underflows for y < ~ -745.1.
```
I think being explicit with the use of "infinity" is clearer than just writing it as "inf" in the docstring.